charts/akeyless-k8s-secrets-injection/templates/_helpers.tpl

```diff
@@ -41,3 +41,12 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+
+{{- define "hpa.api.version" }}
+    {{- if .Capabilities.APIVersions.Has "autoscaling/v2" }}
+        {{- printf "autoscaling/v2" }}
+    {{- else }}
+        {{- printf "autoscaling/v2beta2" }}
+    {{- end }}
+{{- end }}
```

Comment:
We need to make sure this approach works in argocd, something is telling me it won't work

Reply:
why should i care for argocd? this helm is for our customers

Reply:
it's ok just BC for HPA version, worst case it will take a default one if it's not deprecated

Reply:
You should care about argocd because we have customers who use it.
About the default, it's deprecated since kube version 1.23